exchanges/gateio/gateio_wrapper.go

```diff
@@ -2040,3 +2041,102 @@ func (g *Gateio) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) e
 
 	return g.LoadLimits(limits)
 }
+
+// GetFuturesContractDetails returns details about futures contracts
+func (g *Gateio) GetFuturesContractDetails(ctx context.Context, item asset.Item) ([]futures.Contract, error) {
+	if !item.IsFutures() {
+		return nil, futures.ErrNotFuturesAsset
+	}
+	if !g.SupportsAsset(item) {
+		return nil, fmt.Errorf("%w %v", asset.ErrNotSupported, item)
+	}
+	switch item {
+	case asset.Futures:
+		settlePairs := []string{"btc", "usdt", "usd"}
```

Comment:
note: usd returnes nothing.

Reply:
Yeah, I blame gateio. [I figure its best to support all enums that they supply in their API docs than not to](https://www.gate.io/docs/developers/apiv4/en/#list-all-futures-contracts). I can remove it since it technically is unused you're right, but they may add something for it at some point.
[Same applies to the above DeliveryFutures niteroo](https://www.gate.io/docs/developers/apiv4/en/#list-all-futures-contracts-2)